### PR TITLE
fetch-configlet: Update to latest version

### PR DESCRIPTION
**Pull request by guygastineau**
fetch-configlet: new version as of 4 months ago

Hopefully this will solve the travis issue.

Here is the [commit](https://github.com/exercism/configlet/commit/c6324a79289af2bbe2c97c26497ae9293be5ce6c#diff-0cb166cee465745304f179d67aae27d4) that made the update to `exercism/configlet`.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

**Comment by guygastineau** [comment]
Well, Inspecting these scripts there is very little difference between them, and they definitely resulted in the same command being executed, so I don't know hwy it didn't work on my other branch.

I will go ahead and make travis build on that branch again.

File: bin/fetch_configlet [hunk 1]
```diff
@@ -1,32 +1,50 @@
...
-VERSION="$(curl --head --silent $LATEST | awk -v FS=/ '/Location:/{print $NF}' | tr -d '\r')"
-URL=https://github.com/exercism/configlet/releases/download/$VERSION/configlet-$OS-${ARCH}.tgz
 
-curl -s --location $URL | tar xz -C bin/
+VERSION="$(curl --silent --head $LATEST | awk -v FS=/ '/Location:/{print $NF}' | tr -d '\r')"
+URL=https://github.com/exercism/configlet/releases/download/$VERSION/configlet-$OS-${ARCH}.$EXT
+
+case $EXT in
+    (*zip)
+        curl -s --location $URL -o bin/latest-configlet.zip
```
Comment (glennj) [inline bin/fetch_configlet]: nitpick: quote `"$URL"`

Comment (guygastineau) [reply bin/fetch_configlet]: Good point. I actually just merged changes into the configlet repo.

I am doing a PR to clean it up later (including improving the style in the case statements).

We'll hold off on merging this until my next PR goes through over there.

Comment (kotp) [reply bin/fetch_configlet]: Mention that PR here so that there is a reminder there when it gets merged, maybe?

Comment (guygastineau) [reply bin/fetch_configlet]: I didn't have it up yet.

Comment (guygastineau) [reply bin/fetch_configlet]: How do I reference a PR from another repo?

Comment (kotp) [reply bin/fetch_configlet]: https://help.github.com/en/articles/autolinked-references-and-urls

Comment (guygastineau) [reply bin/fetch_configlet]: Thanks for that link @kotp I forgot to do it.

**Comment by guygastineau** [comment]
exercism/configlet#167 is now merged

**Pull request merged**